### DISCUSSION TITLE
hgtcl安装部署失败

### DISCUSSION THREAD

**Discussion by hoocoo** [alibaba/higress#903] [Q&A]
✔️ Installed Deployment:higress-console:higress-system.
Error: failed to install manifests: component higress ApplyManifest err: Deployment in version "v1" cannot be handled as a Deployment: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
failed to install manifests: component higress ApplyManifest err: Deployment in version "v1" cannot be handled as a Deployment: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'

kubectl version
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.8-eks-e16311", GitCommit:"e163110a04dcb2f39c3325af96d019b4925419eb", GitTreeState:"clean", BuildDate:"2020-03-27T22:40:13Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"24+", GitVersion:"v1.24.17-eks-508b6b3", GitCommit:"785e401ddbc8a2879e2d614cf51cbba8a6acca15", GitTreeState:"clean", BuildDate:"2024-01-29T20:59:50Z", GoVersion:"go1.20.10", Compiler:"gc", Platform:"linux/amd64"}

改用helm安装成功

**Comment by CH3CHO** [comment]
kubectl 版本有点低啊，不知道是否有关。
